src/Pages/SignUp.js

```diff
@@ -12,7 +12,11 @@ const SignUp = ({ signup }) => {
     const formData = new FormData(formRef.current);
     const data = Object.fromEntries(formData);
     const userInfo = {
-      user: { email: data.email, password: data.password },
+      user: {
+        email: data.email,
+        username: data.username,
+        password: data.password,
```

Comment:
Does this successfully create a user? Devise usually requires password_confirmation to create a user.

Reply:
It did. Does that mean I need to fix something on the backend?

Reply:
Backend -no, but this form is not handling any password confirmation currently.  Usually this would cause a rollback in your rails server as all of the data was not passed in to properly create an instance.